lib/open_food_network/sales_tax_report.rb

```diff
@@ -57,7 +57,7 @@ def table
           totals = totals_of order.line_items
           shipping_cost = shipping_cost_for order
 
-          [order.number, order.created_at, totals[:items], totals[:items_total],
+          [order.number, order.completed_at, totals[:items], totals[:items_total],
```

Comment:
in this thread: https://github.com/openfoodfoundation/openfoodnetwork/pull/5325/files#r417325953
we agreed with can always use the date format like this:
order.completed_at.strftime("%F %T")
are you ok with this? can you add the formatting?

We also have this problem in lib/open_food_network/order_and_distributor_report.rb
do you mind fixing it there as well?